control-plane/subcommand/server-acl-init/command.go

```diff
@@ -527,6 +531,19 @@ func (c *Command) Run(args []string) int {
 		}
 	}
 
+	if c.flagCreateAPIGatewayToken {
+		apigwRules, err := c.apiGatewayControllerRules()
+		if err != nil {
+			c.log.Error("Error templating api gateway rules", "err", err)
+			return 1
+		}
+		err = c.createLocalACL("api-gateway-controller", apigwRules, consulDC, isPrimary, consulClient)
+		if err != nil {
+			c.log.Error(err.Error())
+			return 1
+		}
+	}
+
```

Comment:
Needs test updates to:
* `TestRun_TokensPrimaryDC`
* `TestRun_TokensReplicatedDC`
* `TestRun_TokensWithProvidedBootstrapToken`

Sorry we should have the "adding a new token" process documented somewhere.